client/components/AddSponsor.tsx

```diff
@@ -29,32 +36,43 @@ const AddSponsor: React.FC = () => {
 
   return (
     <>
-      {(responseMsg || errors) && <ResponseDiv>{responseMsg}</ResponseDiv>}
-      <Form onSubmit={handleSubmit(onSubmit)}>
-        <label>Sponsor Name: </label>
-        <Input
+      {(responseMsg || errors) && <div>{responseMsg}</div>}
```

Comment:
was the `ResponseDiv` margin intentionally left out?

Reply:
Completely forgot about that component, but since this is only a skeleton a simple div should be fine, don't you agree?

Reply:
I'm fine with it, though I don't know what it looks like at the moment :P

Though looking at this a second time I'm not sure what it's supposed to be doing here.  If `responseMsg` is falsy but `errors` is truthy, it'll just render an empty div, and if `responseMsg` is truthy then it doesn't matter what `errors` is because it's not used in the jsx?

Reply:
Honestly I didn't really look at the logic, just changed from styled-components to use materialUI's native JSS styling (or in this case remove it). We should maybe do something like
```jsx
{error ? <h1>Error: {responseMsg}</h1> : <div>{responseMsg}</div>}
```

Reply:
I'm fine with either of them but I think someone else will have better context (whoever implemented that piece)

Reply:
@joelrozen wrote this, any comments, did we miss something?

Reply:
Hmmm... I don't remember fully, but I think perhaps I was going to have `errors` be related to input / frontend errors, but then just kept it as the browser based validation. 
Would make sense, sense `errors` is completely undefined and my errorhandler in submitForm just sets `responseMsg` anyway, could basically just keep it as `responseMsg && <div>{responseMsg}</div>`